### ISSUE TITLE
Any link that's a dropdown is not clickable on desktop

### ISSUE THREAD

**Issue by Lindolini** [viljamis/responsive-nav.js#139]
I'm using the multi-level solution and any parent dropdown link that uses the class "dropdown-toggle" can't be clicked on for desktop. If it's not a dropdown, the link can be clicked. It's the case with the demo as well. Any idea how to fix this?

https://github.com/viljamis/responsive-nav.js/tree/multiple-levels

**Comment by Lindolini** [comment]
Hello,

So after I commented out that chunk of code, the nav links with the class "dropdown" are now clickable (which is what was needed); however, when I get to mobile size, the toggle menu itself is no longer clickable. Is there a way to only target areas using the class "dropdown" but not the whole toggle menu itself?

tl;dr: 
Make .dropdown links clickable without disabling the behavior of the .nav-toggle element.

**Comment by Lindolini** [comment]
Hello,

So I commented out lines 696-697. Everything seems to work how I want it (clickable .dropdown links and tappable .nav-toggle). Would that cause any issues? If not, awesome!

  696      // this._preventDefault(e);
  697      // if (!isMobile) return;

**Comment by coffeeneed** [comment]
Hi. Nice information and research. However, this doesn't solve my problem as I'm using the multiple-levels branch: https://github.com/viljamis/responsive-nav.js/blob/multiple-levels/responsive-nav.css

Something has to be different there, since the +sign and the link it self now are equal on mobile view (it expands the menu). I want the field to be expandable via the +sign and clickable via the linktext.

Do you have any ideas on how to solve this?

**Comment by richardNorthwind** [comment]
any solutions to this yet?

**Comment by Lindolini** [comment]
yea, feel free to inspect the html and css http://focus.mudbugmedia.com/

**Comment by Lindolini** [comment]
#146
